src/poojap/Assignment_47.java

```diff
@@ -0,0 +1,36 @@
+/*Assignment - 47 : 27th Dec'2020
+Replace second last occurance of "Hi" with your name.
+String msg = "Hi globant Hello Hi Hi globant Hi Hello Hello globant Hi Techno Credits Techno";
+output : Hi globant Hello Hi Hi globant Maulik Hello Hello globant Hi Techno Credits Techno */
+
+package poojap;
+
+public class Assignment_47 {
+
+	static void replaceWord(String str, String replaceword) {
+		String[] input = str.split(" ");
+		int cnt = 0;
+		for (int i = input.length - 1; i >= 0; i--) {
+			if (input[i].equals(replaceword)) {
+				cnt++;
+				if (cnt == 2) {
+					input[i] = "Pooja";
```

Comment:
you can pass "pooja" in variable in method parameter, you can also use the substring method to handle this,

https://github.com/KrishnaKTechnocredits/JAVATechnoOct2020/blob/main/src/brijesh/assignment47/ReplaceSecondLastOccurance.java